klib/platform.gradle

```diff
@@ -8,13 +8,15 @@ buildscript {
     }
 }
 
-def taskName = "${gradle.startParameter.projectProperties['name']}"
+def libraryName = "${gradle.startParameter.projectProperties['name']}"
```

Comment:
Please rework to avoid collision with recent Slava's changes.

Reply:
Yep. And Vasya's too.